programs/disks/CommandCopy.cpp

```diff
@@ -59,7 +59,7 @@ class CommandCopy final : public ICommand
         String relative_path_from = validatePathAndGetAsRelative(path_from);
         String relative_path_to = validatePathAndGetAsRelative(path_to);
 
-        disk_from->copy(relative_path_from, disk_to, relative_path_to);
+        disk_from->copyDirectoryContent(relative_path_from, disk_to, relative_path_to);
```

Comment:
This changes the behavior of the copy command.

Before it was `cp foo bar`
After it will do `cp foo bar/foo`

Was that the intention?

Reply:
No, it was not intended, I will fix it. However, it means that we don't have proper tests for `clickhouse-disk` utility, cc: @Varinara, @kssenii

Reply:
Indeed. I've added one in https://github.com/ClickHouse/ClickHouse/pull/51448, that how I found it.